.setup/INSTALL_SUBMITTY_HELPER.sh

```diff
@@ -372,6 +372,17 @@ mkdir -p ${SUBMITTY_INSTALL_DIR}/library
 chown submitty_php:submitty_php ${SUBMITTY_INSTALL_DIR}/library
 chmod g+rwx ${SUBMITTY_INSTALL_DIR}/library
 
+# Create test folders
+mkdir -p ${SUBMITTY_INSTALL_DIR}/library/WE_WANT_TO_SEE_THIS
```

Comment:
Test files can and should be created with the setUp and tearDown functions. I really do not think these should be in the install script.....